osu.Game.Rulesets.Mania/Mods/ManiaMod.cs

```diff
@@ -105,6 +105,8 @@ public void ApplyToRulesetContainer(RulesetContainer<ManiaHitObject> rulesetCont
 
     public abstract class ManiaKeyMod : Mod
     {
+        // TODO: implement using the IApplicable interface. Haven't done so yet because KeyCount isn't even hooked up at the moment.

```

Comment:
Make this into an issue rather than an inline todo please.

Reply:
inline because the class already exists (but isn't implementing it). issue might be missed by future implementor

Reply:
I would say that a comment has more chance to be missed than an issue thread...

Reply:
i'll make an issue as well, but i'd like to leave this here

Reply:
Sounds fair

Reply:
https://github.com/ppy/osu/issues/1833